relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -532,6 +562,43 @@ fn scrub_function(string: &str) -> Option<String> {
     Some(FUNCTION_NORMALIZER_REGEX.replace_all(string, "*").into())
 }
 
+fn scrub_mongodb_query(query: &str, command: &str, collection: &str) -> Option<String> {
+    let original: Value = match serde_json::from_str(query) {
+        Ok(v) => v,
+        Err(_) => return None,
+    };
+
+    let mut scrubbed_map: Map<String, Value> = Map::new();
+    original.as_object()?.iter().for_each(|(key, value)| {
+        scrubbed_map.insert(key.clone(), scrub_mongodb_leaf_nodes(value));
```

Comment:
I think none of these clones are required, you can either move the elements into the new map or modify the map in place. The in-place modifications seems to be the more efficient and easier solution to this.